public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx

```diff
@@ -32,6 +32,9 @@ import {
 } from '../../../utils/requests';
 import TestEditorContextTargeting from '../testEditorContextTargeting';
 import { getDesignForVariant } from '../../../utils/bannerDesigns';
+import { BanditEditor } from '../banditEditor';
+
+const BANDIT_FEATURE_SWITCH = true;
```

Comment:
Do we need this?

Reply:
Removed that as we are going to keep bandit as a permanent feature